app/models/foreman_docker/service/actions/container/provision.rb

```diff
@@ -0,0 +1,26 @@
+module ForemanDocker
+  module Service
+    module Actions
+      module Container
+        class Provision < ::Actions::EntryAction
+          def plan(container)
+            action_subject(container)
+
+            sequence do
+              plan_action(Container::Pull, container)
+              plan_action(Container::Start, container)
+            end
+          end
+
+          def humanized_name
+            _('Provision container')
+          end
+
+          def finalize
+            Rails.logger.info('Finished provisioning of Docker container')
```

Comment:
It's better to use `Action#action_logger` or just adding the info to output if it makes sense (it does not here).